thirdparty/xatlas/xatlas.cpp

```diff
@@ -1955,7 +1955,10 @@ class Mesh
 
 		//make sure its not a degenerate
 		bool degenerate = distanceSquared(base[0], base[1]) < NV_EPSILON || distanceSquared(base[0], base[2]) < NV_EPSILON || distanceSquared(base[1], base[2]) < NV_EPSILON;
-		xaDebugAssert(!degenerate);
+		if(degenerate) {
```

Comment:
There should be a space between the "if" and the parenthesis.

Reply:
Resolved. Thanks.